examples/viewer/viewer.cc

```diff
@@ -625,6 +728,10 @@ static void Draw(const std::vector<DrawObject>& drawObjects,
     glBindTexture(GL_TEXTURE_2D, 0);
   }
 
+  static bool bDrawWireFrame = false;
```

Comment:
Local change. Giving option to draw wire frame. By default, I don't draw the wireframe in order to see the effect of smoothing normals more clearly.